app/src/main/res/values/strings.xml

```diff
@@ -982,4 +982,5 @@ Do you want to switch to this account?</string>
   <string name="choose_your_master_password">Choose your master password</string>
   <string name="choose_a_unique_and_strong_password_to_keep_your_information_safe">Choose a unique and strong password to keep your information safe.</string>
   <string name="minimum_characters">%1$s characters</string>
+  <string name="min_characters">%1$s characters</string>
```

Comment:
Is there a reason we couldn't use the string right above?

Reply:
lol this is a result of committing some changes and stashing others and not remembering where I left off